api/app/tests/testcase/testcases.py

```diff
@@ -0,0 +1,47 @@
+from jsonschema import validate
+from jsonschema.exceptions import ValidationError
+from rest_framework.test import APITestCase
+
+
+class ValidationException(Exception):
+    pass
+
+
+class JsonAPITestCase(APITestCase):
+
+    def assertJsonSchema(self, schema: dict, json_dict: dict):
+        """ Validates json_dict against schema. Schema format as defined on json-schema.org . If
+        additionalProperties is not set in schema, it will be set to False. This assertion
+        treats all properties as 'required', except for when required properties are explicitly
+        set according to the json-schema.org standard """
+
+        if "additionalProperties" not in schema:
+            schema["additionalProperties"] = False
+
```

Comment:
Not necessarily against, but setting `"additionalProperties" = False` seems a job for the schema author --- not of the `JsonAPITestCase`.

Reply:
besproken